### PR TITLE
docs: initial auto-updates and rollbacks pages

### PR DESCRIPTION
**Pull request by lucab**
This adds two initial pages on OS updates, covering:
 * auto-updates
 * manual rollbacks

### CODE REVIEW

File: modules/ROOT/pages/auto-updates.adoc [hunk 1]
```diff
@@ -0,0 +1,20 @@
+= Auto-updates
+
+Fedora CoreOS provides atomic updates and rollbacks via https://ostree.readthedocs.io/en/latest/[OSTree] deployments.
+
+By default, the OS performs continuous auto-updates via two components:
+
+ * https://github.com/coreos/rpm-ostree[rpm-ostree] handles multiple on-disk OSTree deployments, and it can switch between them at boot-time.
+ * https://github.com/coreos/zincati[Zincati] continuously checks for OS updates and applies them via rpm-ostree.
+
+== Updates finalization
```
Comment (ashcrow) [inline modules/ROOT/pages/auto-updates.adoc]: `Update finalization`

Comment (lucab) [reply modules/ROOT/pages/auto-updates.adoc]: Ack, done.

File: modules/ROOT/pages/manual-rollbacks.adoc [hunk 2]
```diff
@@ -0,0 +1,12 @@
+= Manual rollbacks
+
+In case of a misbehaving OS update, it is possible to temporarily or permanently switch back to the previous working OS version.
+This is a manual operation which requires human intervention and console access.
+
+Upon updates, the previous OS deployment is kept on disk so that it can be used as a fallback if something goes wrong.
+
+In order to temporarily boot into the previous OS deployment, hold down `Shift` during OS bootup and select the relevant OS entry in the bootloader menu.
```
Comment (ajeddeloh) [inline modules/ROOT/pages/manual-rollbacks.adoc]: Is this a grub thing? I've never heard of this.

Comment (ashcrow) [reply modules/ROOT/pages/manual-rollbacks.adoc]: It is. [Here's](https://fedoraproject.org/wiki/Changes/HiddenGrubMenu#How_To_Test) an example from a test.

**Comment by lucab** [comment]
(I'd need somebody to merge this for me, as I can't here)

**Comment by ashcrow** [comment]
@coreos/fedora-coreos-docs PTAL for merge

**Pull request merged**